project/Build.scala

```diff
@@ -119,6 +129,38 @@ object MyScalaJSPlugin extends AutoPlugin {
         (fullClasspath in (Build.linker.v2_12, Runtime)).value
       },
 
+      scalaJSLoggerFactory := {
+        val default = scalaJSLoggerFactory.value
+
+        ExposedValues.autoImport.loggerTimingOut.?.value match {
+          case None => default
+
+          case Some(file) =>
+            /* Instrument logger to dump calls to `time` to a file.
+             * The way we manage the file is a hack: It is difficult to properly
+             * manage resources in sbt's settings system, so we simply set the
+             * printStream to autoFlush, avoiding that we have to close the file.
+             */
+            val stream = new PrintStream(new FileOutputStream(file, /*append=*/true), /*autoFlush=*/true)
+
+            (sbtLogger: SbtLogger) => {
+              val base = default(sbtLogger)
+              new Logger {
+                def log(level: Level, message: => String): Unit =
+                  base.log(level, message)
+
+                def trace(t: => Throwable): Unit =
+                  base.trace(t)
+
+                override def time(title: String, nanos: Long): Unit = {
+                  stream.println(s"$title, $nanos")
```

Comment:
If the output is meant to be CSV, shouldn't it be
```suggestion
                  stream.println(s"$title,$nanos")
```
instead?

Reply:
Yes, indeed.